content/changelog/2024-11-29.md

```diff
@@ -0,0 +1,108 @@
+---
+description: Organizations are GA! Also, a new Schema Diff GitHub Action, Postgres version updates, and more.
+---
+
+### Neon Organizations are GA! 🎉
+
+We're happy to announce that Neon **Organizations** is now **generally available** for all everyone looking to move their teams to Neon.
+
+![get started with your new org](/docs/manage/org_projects.png)
+
+Create a new organization, transfer over your projects, invite your team members and get started collaborating. The Organizations feature lets you manage all of your team's projects under a single account — with billing, role management, and project transfer capabilities in one accessible location.
+
+See [Neon Organizations](/docs/manage/organizations) to get started. Organizations is a paid account feature.
+
+## Organization API keys
+
+Along with GA for Organizations, we're also announcing support for [Organizations](https://neon.tech/docs/manage/organizations). As an admin of an organization, you control your organization's API keys. These keys provide admin-level access to all organization resources, including projects, members, and billing information. See [Organization API keys](/docs/manage/orgs-api).
```

Comment:
>Along with GA for Organizations, we're also announcing support for [Organizations](https://neon.tech/docs/manage/organizations).

i don't understand this sentence.

Reply:
Ah, that should have been "Organization API Keys"

Reply:
changed to:
Along with GA for Organizations, we're also announcing support for [Organization API Keys](/docs/manage/orgs-api). As an admin of an organization, you control your organization's API keys. These keys provide admin-level access to all organization resources, including projects, members, and billing information.